addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -242,10 +242,11 @@ class AddPageTemplatePreview extends Component {
             return;
         }
         const wrapEl = this.iframeRef.el.contentDocument.getElementById("wrap").cloneNode(true);
+        const templateId = this.props.template.key;
```

Comment:
The change in this file should be in their own commit IMO, I don't think it has much to do with the processor util creation :+1: